.github/workflows/libs.yml

```diff
@@ -69,3 +69,24 @@ jobs:
         with:
           name: build_logs
           path: macos/build/tmp
+
+linux:
```

Comment:
This part has the wrong indentation level. See [here](https://github.com/weisJ/auto-dark-mode/actions/runs/174221251)

Reply:
Haha. I thought I double checked that! I copied and pasted the Mac OS section and tweaked it a bit. IntelliJ can get finicky with pasting multiple lines of indented YAML.

Reply:
Fixed in 2821755da0577d2abe5f0e7f9a355461baa8dde2.